### DISCUSSION TITLE
[@types/tabulator-tables] - missing correct alias for TabulatorFull

### DISCUSSION THREAD

**Discussion by mbulfair** [DefinitelyTyped/DefinitelyTyped#58012] [Issues with a @types package]
Using  `@types/tabulator-tables": "^5.0.1`

`import { TabulatorFull as Tabulator } from 'tabulator-tables';` 

Following http://tabulator.info/docs/5.0/install#setup-full

When attempting initialize `new Tabulator()` it will throw a error that it is not a function. 

Work around is to put a ts-ignore, which builds and run correctly.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi, I tested it and it seems it works fine.

this works?
```typescript
import { TabulatorFull } from 'tabulator-tables';
new TabulatorFull();
```

**Comment by mike-lischke** [comment]
Still, I should have at least derive TabulatorFull from Tabulator, for code completion info etc. I'll look into it over the next weekend.

**Reply by mike-lischke** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58371 has been merged, so this issue should be solved by now.
